integration_test/gce/gce_testing.go

```diff
@@ -1273,6 +1273,16 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 		return nil, err
 	}
 
+	if isUbuntu(vm.Platform) {
```

Comment:
This problem isn't exclusive to Ubuntu; it could happen on any apt-based distro.